spec/webauthn/relying_party_spec.rb

```diff
@@ -1,11 +1,13 @@
 # frozen_string_literal: true
 
-require "ostruct"
 require "spec_helper"
 require "webauthn/fake_authenticator"
 require "webauthn/fake_client"
 require "webauthn/relying_party"
 
+Credential = Struct.new(:webauthn_id, :public_key, :sign_count, keyword_init: true)
```

Comment:
What do you think about defining it as a `let` instead so that we don't create global constants?

```suggestion
let(:credential_klass) { Struct.new(:webauthn_id, :public_key, :sign_count, keyword_init: true) }
```

Reply:
That makes sense!